common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -138,6 +149,19 @@ private void scanCharacterInfoPage() {
         query.executeQuery();
     }
 
+    private void scanCratesBombsCosmeticsPage() {
+        ScriptedContainerQuery query = ScriptedContainerQuery.builder("Crates, Bombs & Cosmetics Query")
+                .onError(msg -> WynntilsMod.warn("Error querying Crates, Bombs & Cosmetics: " + msg))
+
+                // Open /use menu
+                .then(QueryStep.sendCommand("use")
+                        .expectContainerTitle("Crates, Bombs & Cosmetics")
```

Comment:
extract this to a final variable. I am not sure if this class should hold it though.

Reply:
The command or the container title? If the container title, do you also want me to move the existing scanCharacterInfoPage() one?

Reply:
Title. and yes